src/anonymousUserTracking/anonymousUserEventManager.ts

```diff
@@ -366,6 +352,40 @@ export class AnonymousUserEventManager {
     }
   }
 
+  private async storeUserUpdateToLocalStorage(
+    newDataObject: Record<
+      any /* eslint-disable-line @typescript-eslint/no-explicit-any */,
+      any /* eslint-disable-line @typescript-eslint/no-explicit-any */
+    >
```

Comment:
Do we really have no idea what the data obj is supposed to look like?

Reply:
(here and lines 314-317)

Reply:
I will type that out 👍